processor/src/trace/mod.rs

```diff
@@ -45,6 +45,9 @@ pub struct AuxTraceHints {
     pub(crate) chiplets: ChipletsAuxTraceBuilder,
 }
 
+// TODO: add documentation
+pub type MainTrace = ColMatrix<Felt>;
```

Comment:
The `MainTrace` that the issue was referring to was [this one](https://github.com/0xPolygonMiden/miden-vm/blob/2d417030b7d248991a3a1431510f73fd8f59d843/air/src/trace/main_trace.rs#L31-L33) (it's on the `al-simplify-aux` branch)

Reply:
got it thanks